covalent_ssh_plugin/ssh.py

```diff
@@ -70,6 +70,9 @@ class SSHExecutor(RemoteExecutor):
         do_cleanup: Whether to delete all the intermediate files or not
     """
 
+    max_connection_attempts = 5
+    retry_wait_time = 5  # seconds
```

Comment:
I think it'll be better if these were constructor arguments instead of class attributes. Might make things easier if the user wants to change these values.